_posts/2023-02-07-homebrew-4.0.0.md

```diff
@@ -0,0 +1,53 @@
+  - [`HOMEBREW_INSTALL_FROM_API` is now the default behaviour so the variable has been removed and is a no-op](https://github.com/Homebrew/brew/pull/14412).
+  - If you wish to opt-out of this behaviour change, you can [`export HOMEBREW_NO_INSTALL_FROM_API=1`](https://github.com/Homebrew/brew/pull/14375)
+  - This behaviour is automatically disabled when using [certain commands](https://github.com/Homebrew/brew/pull/14524) or in configurations that [mostly build from source](https://github.com/Homebrew/brew/pull/14480).
+  - [`HOMEBREW_API_DOMAIN` can be set to use mirrors for](https://github.com/Homebrew/brew/pull/14494) [formulae.brew.sh](https://formulae.brew.sh).
+- Work has begun to migrate from Google Analytics to a self-hosted InfluxDB instance. You can [help us test this beta feature by setting `export HOMEBREW_ANALYTICS_ENABLE_INFLUX=1`](https://github.com/Homebrew/brew/pull/13356).
+- [macOS `.pkg` files are generated by releases](https://github.com/Homebrew/brew/pull/14265). You can help us test this beta feature by [downloading the generated package artifact from the relevant GitHub Actions release events](https://github.com/Homebrew/brew/actions/workflows/build-pkg.yml?query=event%3Arelease).
+- [The `homebrew/ubuntu16.04:master` image has been deprecated](https://github.com/Homebrew/brew/pull/13819).
+- [Various major release deprecations and disables](https://github.com/Homebrew/brew/pull/14382).
+
+
+Other changes since 3.6.0 I'd like to highlight are the following:
+
+- [`brew test` sets `PYTHONDONTWRITEBYTECODE` to improve performance.](https://github.com/Homebrew/brew/pull/14338).
+- [VSCode devcontainers are available in the Homebrew/brew repository for use with e.g. GitHub Codespaces](https://github.com/Homebrew/brew/pull/14027).
+- [`brew install --adopt` allows taking ownership of existing installed casks](https://github.com/Homebrew/brew/pull/14033).
+- [`brew install --dry-run` allows viewing what `brew install` will do before it is run](https://github.com/Homebrew/brew/pull/13706).
+- [`brew docs` opens](https://github.com/Homebrew/brew/pull/13844) [docs.brew.sh](https://docs.brew.sh).
+- [`--cask` is never required on Linux](https://github.com/Homebrew/brew/pull/14206).
+- [`service do` blocks allow defining a `run` command per platform](https://github.com/Homebrew/brew/pull/14371).
+- [`brew install` uses the local cache while installing dependencies from `pip`](https://github.com/Homebrew/brew/pull/14239).
+- [`brew doctor` no longer complains about BitDefender](https://github.com/Homebrew/brew/pull/14300).
+- [`brew install` will also suggest casks rather than just formula when it fails to find the requested package](https://github.com/Homebrew/brew/pull/14240).
```

Comment:
```suggestion
- [`brew install` will also suggest casks rather than just formulae when it fails to find the requested package](https://github.com/Homebrew/brew/pull/14240).
```